spec/veriff/session_spec.rb

```diff
@@ -49,6 +50,106 @@ module Veriff
       end
     end
 
+    describe '#watchlist_screening' do
+      let(:parsed_response) do
+        {
+          status: 'success',
+          data: {
+            attempt_id: "aea9ba6d-1b47-47fc-a4fc-f72b6d3584a7",
+            vendor_data: nil,
+            check_type: "updated_result",
+            match_status: "possible_match",
+            search_term: {
+              name: "Mirko Kokki",
+              year: "1960"
+            },
+            total_hits: 1,
+            created_at: "2021-07-05T13:23:59.851Z",
+            hits: [{
+              matched_name: "Miro kokkino",
+              countries: [
+                "Australia",
+                "Brazil"
+              ],
+              date_of_birth: "1963",
+              date_of_death: nil,
+              match_types: [
+                "aka_exact",
+                "year_of_birth"
+              ],
+              aka: [
+                "Mirkoni kokki",
+                "Mirkor Kokki"
+              ],
+              associates: [
+                "Desmon Lamela",
+                "Fred Austin"
+              ],
+              listings_related_to_match: {
+                warnings: [{
+                  source_name: "FBI Most Wanted",
+                  source_url: "http://www.fbi.gov/wanted",
+                  date: nil
+                }],
+                sanctions: [{
+                  source_name: "Argentina Ministerio de Relaciones Exteriores y Culto Sanciones de la ONU",
+                  source_url: "https://www.cancilleria.gob.ar/es/politica-exterior/seguridad-internacional/comite-de-sanciones",
+                  date: nil
+                }],
+                fitness_probity: [],
+                pep: [{
+                  source_name: "United Kingdom Insolvency Service Disqualified Directors",
+                  source_url: "https://www.insolvencydirect.bis.gov.uk/IESdatabase/viewdirectorsummary-new.asp",
+                  date: nil
+                }],
+                adverse_media: [{
+                  source_name: "SNA's Old Salt Award Passed to Adm. Davidson",
+                  source_url: "https://www.marinelink.com/amp/news/snas-old-salt-award-passed-adm-davidson-443093",
+                  date: nil
+                }]
+              }
+            }]
+          }
+        }
+      end
```

Comment:
IMHO it will be a good option to add also some bad or empty responses to tests. What do you think?

Reply:
Writing these tests I kept the current convention. 

In addition, we have to remember that it is just an API wrapper. Regardless of the given response from Veriff we just return the request body and let the application (BFID) handle it. WDYT? Otherwise, could you clarify what examples do you mean?